src/randomPortChromeConnection.ts

```diff
@@ -0,0 +1,53 @@
+import { chromeConnection } from 'vscode-chrome-debug-core';
+import { utils, chromeUtils } from "vscode-chrome-debug-core";
+import { logger } from "vscode-chrome-debug-core";
+import * as errors from './errors';
+
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+/**
+ * Chrome connection class that supports launching with --remote-debugging-port=0 to get a random port for the debug session
+ */
+export class RandomPortChromeConnection extends chromeConnection.ChromeConnection {
```

Comment:
I'd rename this to ConnectionWithChromeProvidedPort or ConnectionWithChromeChoosenPort

Reply:
done